tests/Common/JobsListTest.php

```diff
@@ -34,6 +35,21 @@ public function testJobsListing(): void
         $twoJobs = $this->_client->listJobs(['limit' => 2]);
         $this->assertEquals(2, count($twoJobs));
 
+        $errorJobs = $this->_client->listJobs(['storage_job_status' => ['error']]);
+        $this->assertEquals(6, count($errorJobs));
+
+        $errorJobs = $this->_client->listJobs(['storage_job_status' => ['success']]);
+        $this->assertEquals(20, count($errorJobs));
+
+        $startBefore = $this->_client->listJobs(['start_time_to' => (new DateTime('now'))->format('Y-m-d H:i:s')]);
+        $this->assertEquals(20, count($startBefore));
+
+        $startBefore = $this->_client->listJobs([
+            'start_time_from' => (new DateTime('-1 hour'))->format('Y-m-d H:i:s'),
+            'start_time_to' => (new DateTime('now'))->format('Y-m-d H:i:s'),
+            ]);
+        $this->assertEquals(12, count($startBefore));
```

Comment:
Na tos přišel jak na ty čísla? To přeca nebude stabilní kdekoliv jinde to nebude sedět taky bude záležek kdy ten test spustíš. Nenapadá mě způsob jak bys to zaručil.
Jediná možnost je imho vylistovat to vždycky s limitem a zkontrolovat že sedí kritéria.

Reply:
Vždyť jsem ti říkal na SU, že testy k tomu ještě budu dodělávat, ať se prosím koukneš, jestli ti dává smysl ta implementace a dokumentace.

No a ty čísla jsou random podle mého localu a ano, není to stabilní :-D

Reply:
Už tu je smysluplný test - aspoň jen co to šlo